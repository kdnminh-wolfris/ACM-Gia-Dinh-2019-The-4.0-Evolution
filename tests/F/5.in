8354 2101 1235 2627