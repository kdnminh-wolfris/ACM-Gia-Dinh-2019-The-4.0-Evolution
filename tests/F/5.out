7119 8354